### ISSUE TITLE
Add cross support for TensorFlow and PyTorch backends

### ISSUE THREAD

**Issue by diegofiori** [nebuly-ai/nebullvm#66]
Now that we have implemented backends that work natively on PyTorch and TF, we could exploit them further by allowing TF models to be converted to Torch models and vice versa. In this way, we will take advantage of situations where one is faster than the other in performing particular computations.
